app/Position.php

```diff
@@ -30,11 +30,20 @@ class Position extends Model
 
     public function getTypeAttribute()
     {
-        return self::types[$this->attributes['type']][0];
+        return self::TYPES[$this->attributes['type']][0];
     }
 
     public function getTypeHumanAttribute()
     {
-        return self::types[$this->attributes['type']][1];
+        return self::TYPES[$this->attributes['type']][1];
+    }
+
+    public function getSuffixAttribute()
+    {
+        static $GROUP_1 = 1;
+        $matches = [];
+        preg_match("/_([A-Z]*+$)/", $this->attributes['callsign'], $matches);
```

Comment:
How will this work with the `TMA` position type? I.E will it effect logic down the line that this returns TMA (not one of the approved suffixes?)

Reply:
I assume you mean things like DEP?

Reply:
Not really, as we don't have those positions in the UK (at the moment at least) - checking the live Core database, this position type is assigned to MAN/STC/LTC_CTR positions so should be fine as long as we don't deviate from that!

Reply:
Also, see below comment @AxonC

Reply:
> Also, constants vs library for the class?
> 
> As we have logic in there I would have thought library more appropriate for the namespace

See I'm not sure if it is a library as such since I tend to refer to those as third party. I just didn;'t know where was best to put it in the folder structure. Maybe a service? I'm not sure.